src/Service.Host/client/src/components/ChangeRequests/ChangeRequest.js

```diff
@@ -35,20 +41,82 @@ function ChangeRequest() {
         itemSelectorHelpers.getItem(reduxState.changeRequestStatusChange)
     );
 
+    const changedState = (changereq, origreq) => {
+        if (changereq?.changeState !== origreq?.changeState) {
+            return true;
+        }
+
+        if (
+            changereq.bomChanges.filter(c => c.changeState === 'CANCEL').length >
+            origreq.bomChanges.filter(c => c.changeState === 'CANCEL').length
+        ) {
+            return true;
+        }
+
+        if (
+            changereq.pcasChanges.filter(c => c.changeState === 'CANCEL').length >
+            origreq.pcasChanges.filter(c => c.changeState === 'CANCEL').length
+        ) {
+            return true;
+        }
+
+        if (
+            changereq.bomChanges.filter(c => c.changeState === 'LIVE').length >
+            origreq.bomChanges.filter(c => c.changeState === 'LIVE').length
+        ) {
+            return true;
+        }
+
+        if (
+            changereq.pcasChanges.filter(c => c.changeState === 'LIVE').length >
+            origreq.pcasChanges.filter(c => c.changeState === 'LIVE').length
+        ) {
+            return true;
+        }
+
+        return false;
```

Comment:
this big if sequence followed by a default false return could just be rewritten as 

`const changedState => condition1
 || condition2 
|| condition3 etc `

I'm not sure if thats better really. More concise but maybe not as clear? Up to you

Reply:
i tried it and it was shorter but denser.  think i prefer the longer more descriptive version